doc/start/hardware-recommendations.rst

```diff
@@ -54,55 +54,60 @@ Monitors and managers (ceph-mon and ceph-mgr)
+for peak usage. For very small clusters, 32 GB suffices. For clusters of up to,
+say, 300 OSDs go with 64GB. For clusters built with (or which will grow to)
+even more OSDs you should provision 128GB. You may also want to consider
+tuning the following settings:
+
+* :confval:`mon_osd_cache_size`
+* :confval:`rocksdb_cache_size`
+
 
 Metadata servers (ceph-mds)
 ---------------------------
 
 The metadata daemon memory utilization depends on how much memory its cache is
 configured to consume.  We recommend 1 GB as a minimum for most systems.  See
-``mds_cache_memory``.
+:confval:`mds_cache_memory_limit`.
 
 
 Memory
 ======
 
 Bluestore uses its own memory to cache data rather than relying on the
 operating system page cache.  In bluestore you can adjust the amount of memory
-the OSD attempts to consume with the ``osd_memory_target`` configuration
+the OSD attempts to consume with the :confval:`osd_memory_target` configuration
 option.
 
-- Setting the osd_memory_target below 2GB is typically not recommended (it may
-  fail to keep the memory that low and may also cause extremely slow performance.
+- Setting the :confval:`osd_memory_target` below 2GB is typically not
+  recommended (it may fail to keep the memory that low and may also cause
+  extremely slow performance.
 
 - Setting the memory target between 2GB and 4GB typically works but may result
-  in degraded performance as metadata may be read from disk during IO unless the
+  in degraded performance: metadata may be read from disk during IO unless the
   active data set is relatively small.
 
-- 4GB is the current default osd_memory_target size and was set that way to try
-  and balance memory requirements and OSD performance for typical use cases.
+- 4GB is the current default :confval:`osd_memory_target` size.  This default
+  was chosen for typical use cases, and is intended to balance memory
+  requirements and OSD performance for typical use cases.
 
-- Setting the osd_memory_target higher than 4GB may improve performance when
-  there are many (small) objects or large (256GB/OSD or more) data sets being
-  processed.
+- Setting the :confval:`osd_memory_target` higher than 4GB can improve
+  performance when there many (small) objects or large (256GB/OSD or more) data
+  sets are processed.
 
-.. important:: The OSD memory autotuning is "best effort".  While the OSD may
+.. important:: OSD memory autotuning is "best effort". Although the OSD may
    unmap memory to allow the kernel to reclaim it, there is no guarantee that
-   the kernel will actually reclaim freed memory within any specific time
-   frame.  This is especially true in older versions of Ceph where transparent
-   huge pages can prevent the kernel from reclaiming memory freed from
+   the kernel will actually reclaim freed memory within a specific time
+   frame. This is especially true in older versions of Ceph where transparent
+   huge pages can prevent the kernel from reclaiming memory that was freed from
```

Comment:
Hmmmmm I wonder if this was a factor in my XFS `-n size=65536` ordeal back in the day.  No action here, just talking to myself, move along nothing to see.